static/sass/_patterns_matrix-object.scss

```diff
@@ -0,0 +1,57 @@
+// Based on the matrix component https://github.com/vanilla-framework/vanilla-framework/blob/develop/scss/_patterns_matrix.scss
+@mixin snapcraft-p-matrix-object {
+  $matrix-img-width: map-get($icon-sizes, thumb--small);
+  $matrix-img-gutter: $sph-inter;
+
+  .p-matrix-object {
```

Comment:
It's a bit confusing that it has `matrix` in the name while not being used in matrix pattern at all.

I think we should consider it as a variant of media object pattern and call it accordingly, something like `media-object--stacked` or something? Or just `media-object--snap` if it only fits into our specific use case.

Reply:
dun